readme.md

```diff
@@ -45,7 +45,8 @@ Milestones that we hope to hit in the coming weeks:
 | --------------------------------------|--- | --------|
 | Directly Fund an (Alice, Bob) Channel | mock | ✅ |
 | Virtually Fund an (Alice, Bob) Channel through a single Intermediary | mock | ✅ |
-| Virtually Fund an (Alice, Bob) and an (Alice, Brian) Channel through a single Intermediary | mock | 🟡|
+| Virtually Fund an (Alice, Bob) and an (Alice, Brian) Channel through a single Intermediary | mock | ✅ |
+| Virtually Defund an (Alice, Bob) Channel through a single Intermediary | mock | ✅ |
 | Stream micropayments fron Alice to Bob | mock | |
```

Comment:
Should we drop this from the readme for now?

Reply:
I guess we aren't 100% sure if we `go-nitro` clients will actually be responsible for payments or not. So we can drop this until we decide on that.